tests-integration/tests/http.rs

```diff
@@ -814,6 +814,7 @@ auto_flush_interval = "30m"
 enable_experimental_write_cache = false
 experimental_write_cache_path = ""
 experimental_write_cache_size = "512MiB"
+experimental_write_cache_ttl = "1h"
```

Comment:
Could you please also add them to our config example? Thanks!
https://github.com/GreptimeTeam/greptimedb/tree/main/config